addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -124,8 +122,8 @@ const MoveArticleToDialog = Dialog.extend({
                             children: data.results.map(record => {
                                 return {
                                     id: record.id,
-                                    icon: record.icon,
-                                    text: record.name,
+                                    text: record.display_name,
+                                    subject: record.root_article_id ? record.root_article_id[1] : false,
```

Comment:
Do we want false ? Isn't a void string better ?

Reply:
I realized that this is not required since root_article_id should always have a value